client/src/business-logic/initialize-state.js

```diff
@@ -3,5 +3,5 @@
 import { load } from '../data-access/load.js';
 
 export const initializeState = async () => {
-  await load('../../../data/numbers.json', import.meta);
+  await load('../../../data/quiz.json', import.meta);
```

Comment:
Can you check the path in here with template branch? This was old file. I changed path.

Reply:
Yes Tugba,  it is like Samir did today in class.  Can I merge it?

Reply:
AA, yes you are right.

Reply:
But I realized that @RosaMoran made pr from template to master? Is it correct?

Reply:
I think is correct.

Reply:
@yildiztugba  yes i  made the `PR` you can see it in `template branch` and also the path for  `line 6` of the file `initialize-state.js`  it same that @samir wrote, today i'll start to in UI of `selecctio` part if someone wants help me, that can be great 
have a nice day ladies

Reply:
ooh i forgot, this part i'll put it inside of  `data access` 
i'll make a push from the branch `02-startquiz`